HttpMoq/QueryStringHelper.cs

```diff
@@ -9,6 +9,11 @@ internal static class QueryStringHelper
         {
             var args = new Dictionary<string, string[]>();
 
+            if (queryString == null)

```

Comment:
Can we combine this `if statement` with the one below which checks the length?

Reply:
Sure, have updated this now 👍🏻